### PR TITLE
Add tokenlist CI

### PR DESCRIPTION
**Pull request by dovbrand**
Created CI to create tokenlist

### CODE REVIEW

**Review by connorch — CHANGES_REQUESTED** [review]
Awesome that you got this working!

Danny requested that we also include the data for `chain-information.json` here (from my other [PR](https://github.com/ava-labs/subnet-assets/pull/30)).  We might need to update the data shape to account for this.  Not sure if Danny needs this shape in any particular way, but could be better to structure it like so:

```
{
  43114: {
    chainInfo: {...}
    tokens: [...] 
  }
}
```

Let's collect feedback from Danny on this before merging too.

File: package.json [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+    "author": "Dov Brand <dov.brand@avalabs.org>",
+    "license": "MIT",
+    "type": "module",
   "scripts": {
     "format": "node format.mjs"
   },
   "devDependencies": {
-    "@ethersproject/address": "5.6.0"
+    "@ethersproject/address": "5.6.0",
+    "@types/node": "^17.0.31"
```
Comment (dhrubabasu) [inline package.json:14]: I'd be interested in making this a typescript repo and run the files via https://github.com/lukeed/tsm#usage

File: package.json [hunk 2]
```diff
@@ -1,8 +1,16 @@
 {
+  "name": "subnet-assets",
+    "version": "1.0.0",
+    "description": "repo containing info for subnets in the Avalanche ecosystem",
+    "repository": "git@github.com:ava-labs/subnet-assets.git",
+    "author": "Dov Brand <dov.brand@avalabs.org>",
+    "license": "MIT",
+    "type": "module",
```
Comment (dhrubabasu) [inline package.json:8]: We shouldn't fill out this stuff I think, makes it seem like we're going to publish this as an npm package which we will not do

Comment (dhrubabasu) [reply package.json:8]: Should add `"private": true`

Comment (dovbrand) [reply package.json]: Done

File: package.json (hunk 1 above)
Comment (dovbrand) [reply package.json:9]: What would that do?

Comment (dhrubabasu) [reply package.json:9]: All our .mjs or .js scripts could be .ts files since tsm automagically runs them via ts-node

Comment (dovbrand) [reply package.json:9]: what does that require besides installing the dependency?

Comment (dovbrand) [reply package.json]: Done

**Review by connorch — APPROVED** [review]
lgtm!

**Pull request merged**